src/plugins/data-card/data-card-content.ts

```diff
@@ -33,7 +33,8 @@ export const DataCardContentModel = TileContentModel
   .named("DataCardTool")
   .props({
     type: types.optional(types.literal(kDataCardTileType), kDataCardTileType),
-    caseIndex: 0
+    caseIndex: 0,
+    selectedSortAttributeName: 'none'
```

Comment:
I think you should use `""` or `undefined` here instead of `'none'`. That's one fewer hard coded strings to keep track of.

Reply:
Thanks - I've been working on this.  I ended up using `types.maybe(types.string)` and then I can test for undefined and "none" if the user has switched it back.

Reply:
...I mean I guess I could try putting nothing in the `value` of the `option`...maybe I'll move this to slack

Reply:
that worked, see next push...